src/main/java/xyz/aadev/generitech/common/network/messages/power/PacketSides.java

```diff
@@ -0,0 +1,79 @@
+package xyz.aadev.generitech.common.network.messages.power;
+
+import io.netty.buffer.ByteBuf;
+import net.darkhax.tesla.api.ITeslaConsumer;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'net.darkhax.tesla.api.ITeslaConsumer'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AUselessImportCheck)